HR_allfuncs.py

```diff
@@ -118,149 +122,155 @@ def HR_peakdetect(data_array):
     return time_array
 
 
-def instHR(t_array):
-    """
-            .. function:: instHR(t_array)
-            Calculates instant heart rate from peak time array
-            :param t_array: a 1-d list with time values of peaks
-    """
-    dt_first_beat = t_array[2] - t_array[1]
 
-    instHR = 1/dt_first_beat * 60
 
-    return instHR
 
+class Vitals:
 
 
+    def __init__(self):
+        self.avg_hr_val = float
+        self.inst_hr_val = float
 
+    def hr_averaging(self, averaging_time, time_array):
+        num_arg = 3
+        min_to_sec = 60
 
-def hr_averaging(averaging_time,time_array):
 
-    """
-    .. function:: hr_averaging(averaging_time, tachy_limit = 100, brachy_limit = 60)
+        """
```

Comment:
The docstring should be right after the function definition.

Reply:
done